src/librustc/hir/mod.rs

```diff
@@ -1059,6 +1071,12 @@ pub enum Expr_ {
     /// For example, `[1; 5]`. The first expression is the element
     /// to be repeated; the second is the number of times to repeat it.
     ExprRepeat(P<Expr>, BodyId),
+
+    /// A suspension point for generators
```

Comment:
It would be good to give an example of Rust syntax; this corresponds to a `yield`, right?